hr_recruitment_security/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Hr recruitment security",
+    "version": "12.0.1.0.0",
+    "category": "Human Resources/Recruitment",
```

Comment:
`hr_recruitment` has "Human Resources" as category:

https://github.com/odoo/odoo/blob/14415a0fadafa4a1104693ded33bc53f095b3a65/addons/hr_recruitment/__manifest__.py#L7

so please use the same